rpc/src/rewards_recorder_service.rs

```diff
@@ -23,6 +23,10 @@ pub struct RewardsRecorderService {
     thread_hdl: JoinHandle<()>,
 }
 
+// ReplayStage sends a new item to this service for every frozen Bank. Banks
+// are frozen every 400ms at steady state so checking every 100ms is sufficient
+const TRY_RECV_INTERVAL: Duration = Duration::from_millis(100);
```

Comment:
we could consider deriving this from target slot time

Reply:
I considered this; however, we have a bit of a cycle that made would involve some refactoring:
- Target slot time would come from a `Bank`
- This service is started with some other RPC services before we have a `Bank` accessible because some of the RPC related items are passed to 
https://github.com/anza-xyz/agave/blob/b3c2274bc87390aca04ff941858cf010db32f29f/core/src/validator.rs#L1997-L2029

I think I could technically break this specific service out and start it later so that a `Bank` is available.

However, things would be much cleaner if we did this item that has been a nice-to-have-but-very-low-priority item on my list for quite some time:
https://github.com/solana-labs/solana/issues/25577

If we did this, then `load_bank_forks()` would always unpack a snapshot, and never have replay a block (block 0). Thus, we wouldn't need `cache_block_meta_sender` and other services as inputs.

So, I'm inclined to leave this as-is for now